x/supplier/types/message_stake_supplier_test.go

```diff
@@ -469,6 +507,34 @@ func TestMsgStakeSupplier_ValidateBasic(t *testing.T) {
 			},
 			expectedErr: ErrSupplierInvalidServiceConfig,
 		},
+		{
+			desc: "invalid service configs - owner cannot update",
+			msg: MsgStakeSupplier{
+				Signer:          ownerAddress,
+				OwnerAddress:    ownerAddress,
+				OperatorAddress: operatorAddress,
+				Stake:           &sdk.Coin{Denom: pocket.DenomuPOKT, Amount: math.NewInt(100)},
+				Services: []*sharedtypes.SupplierServiceConfig{
+					{
+						ServiceId: "svcId1",
+						Endpoints: []*sharedtypes.SupplierEndpoint{
+							{
+								Url:     "http://localhost:8081",
+								RpcType: sharedtypes.RPCType_JSON_RPC,
+								Configs: make([]*sharedtypes.ConfigOption, 0),
+							},
+						},
+						RevShare: []*sharedtypes.ServiceRevenueShare{
+							{
+								Address:            sample.AccAddress(),
+								RevSharePercentage: 100,
+							},
+						},
+					},
+				},
+			},
+			expectedErr: ErrSupplierInvalidServiceConfig,
+		},
 	}
```

Comment:
Please double check the test coverage to make sure we cover all the cases.

One of these small operator/owner mishaps usually results in a P0 release we need to push to MainNet.